doc/LibcurlTransportAdapter.md

```diff
@@ -0,0 +1,162 @@
+# Azure SDK for C++ - Libcurl Transport Adapter
+
+## Azure SDK for C++
+
+The [azure SDK for C++](https://github.com/Azure/azure-sdk-for-cpp) enables and easy and consistent experience for consuming Azure services.
+Azure services offer customers a simple way to perform many kinds of operations online. As such, the operations work as a client - server communication where the operations are requested by a client (customer) and executed by a server (Azure server). Communication goes from one point to another, typically through a network.
```

Comment:
```suggestion
Azure services offer customers a simple way to perform many kinds of operations online. As such, the operations work as a client/ erver communication where the operations are requested by a client (customer) and executed by an Azure service over a network.
```

Reply:
Nit: Typo in client/server (written as client/erver).